packages/ui/docs/migration/Stack.stories.mdx

```diff
@@ -0,0 +1,34 @@
+import { Meta } from '@storybook/addon-docs';
+import Link from "@cloudscape-design/components/link";
+import SpaceBetween from "@cloudscape-design/components/space-between";
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
+
+<Meta title="Migration/Stack" />
+
+# Migrating from NorthStar v1 - Stack
+
+Use <Link href='https://cloudscape.design/components/space-between' target='_blank'>Cloudscape SpaceBetween component </Link>
+
+## Update import
+
+```diff
+- import Stack from 'aws-northstar/layouts/Inline';
++ import SpaceBetween from '@cloudscape-design/components/space-between';
+```
+
+## Update code
+
+```diff
+- <Stack spacing='s'>
++ <SpaceBetween direction="vertical" size="s">
+    <Placeholder/>
+    <Placeholder/>
+    <Placeholder/>
+-  </Stack>
++ </SpaceBetween>
+```
+
+
+export const Component = () => {
```

Comment:
SHouldn't there be more components to migrate?

Reply:
Thanks. All the documentations related will be added in another PR.